### DISCUSSION TITLE
Tuya TS011F unable to connect back to Zigbee2MQTT

### DISCUSSION THREAD

**Discussion by Patricen1** [Koenkk/zigbee2mqtt#22461] [Devices]
Hello,

For some reason I do have a Tuya TS011F smart plug (brand new) that went offline within Zigbee2mqtt frontend.
I deleted the device from zigbee2mqtt, pressed "permit join" and long pressed on plug button until the LED blinks. After 3 minutes, LED stops blinking, probably timeout, but device is not associated.
Same issue with another TS011F smart plug that I could never associate to Zigbee2mqtt.

Any help?

Thanks

Notes:
- I'm using Zigbee2mqtt 1.37.0 but the issue was already present using V1.36.0 or V1.36.1
- I do have already 13 Tuya TS011F smart plugs connected, i'm then sure to know what is the accociation process.
- I tried to associate the device close to the coordinator (ConBee2/RaspBee2)
- There are 45 devices connected
- Is there a way to factory reset the smart plugs?
- Is there a cache file where the plugs could be "remembered" within Zigbee2mqtt while the device does not appear on devices list?
- Blocklist and passlist are empty, 
- Edit:
I also tried to allow device to join using specific routers else than coordinator, no improvement.
I do not know how count how many devices are connected to coordinator else than counting links on map.

**Comment by peterthepeter** [comment]
I have the same Problem, do you found a solution for this Problem?
My device disappear after around 1 Hour (brand new) and the i never come back online. Reset does not help.

All my other Z2M are working without any problems (testet with 4 new Sockets)

**Comment by jeroenkl** [comment]
Same issue, no solution found yet
